server/mailet/integration-testing/src/test/java/org/apache/james/transport/mailets/ICSAttachmentWorkflowTest.java

```diff
@@ -20,9 +20,10 @@
 package org.apache.james.transport.mailets;
 
 import static org.assertj.core.api.Assertions.assertThat;
-
+import java.util.Collections;
```

Comment:
I'm pretty sure your import ordering is wrong.

Reply:
Changed by you. Thanks